beet/resources/pack_format_registry.json

```diff
@@ -0,0 +1,5112 @@
+[
+  {
+    "id": "1.21.6",
+    "name": "1.21.6",
+    "release_target": null,
```

Comment:
This is a pretty large file and has a lot of unused data. Do we want to try to generate this file (in github actions?) and preprocess it to only keep the relevant fields?